app/views/notifiers/course/video_notifier/opening/course_notifications/email.html.slim

```diff
@@ -0,0 +1,10 @@
+- video = @object
+- course = video.course
+- host = course.instance.host
+
+- message.subject = t('.subject', course: course.title, video: video.title)
+
+/ TODO: Re-enable when routes are updated
```

Comment:
Not too sure, but I wouldn't want to leave this in the notifiers until I add the controllers action and update `routes.rb`.